app/views/miq_ae_customization/_dialog_field_form.html.haml

```diff
@@ -48,6 +48,30 @@
           &nbsp;
           = _('Only 1 Date or Date/Time element can be present in a Dialog')
 
+    .form-group
+      %label.col-md-2.control-label
+        = _('Auto refresh')
+      .col-md-8
+        = check_box_tag('field_auto_refresh', '1',
+                        @edit[:field_auto_refresh],
+                        'data-miq_observe_checkbox' => {:url => url}.to_json)
```

Comment:
This one only pertains to dynamic fields, there's no way to auto-refresh something that's not dynamic. Also, this form-group should be in the options section. I think this form-group needs to be moved to the `_dialog_field_form_dynamic_options.html.haml` file.